qiskit_nature/mappers/second_quantization/bravyi_kitaev_mapper.py

```diff
@@ -0,0 +1,180 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2020, 2021.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""The Bravyi-Kitaev Mapper."""
+
+import numpy as np
+
+from qiskit.opflow import PauliSumOp
+from qiskit.quantum_info.operators import Pauli
+
+from qiskit_nature.operators.second_quantization.particle_op import ParticleOp
+from qiskit_nature.operators.second_quantization.fermionic_op import FermionicOp
+
+from .qubit_mapper import QubitMapper
+
+
+class BravyiKitaevMapper(QubitMapper):
```

Comment:
As discussed offline, please add a `FermionicQubitMapper` base class (or `FermionicMapper` is also fine imo)